app/assets/stylesheets/about.scss

```diff
@@ -2,7 +2,7 @@
   box-sizing: border-box;
 }
 
-.row {
+.row-about {
```

Comment:
Shouldn't this be 

.about {
  .row{
  }
}

Reply:
No, row-about is a single class I defined to restrict changes specific to the about page only